src/js/editor/key-commands.js

```diff
@@ -76,6 +94,13 @@ export const DEFAULT_KEY_COMMANDS = [{
     }
   }
 }, {
+  str: 'META+SHIFT+LEFT',
+  run(editor) {
+    if (Browser.isMac) {
```

Comment:
Not a blocker for this PR, but we've really got to figure out a more generic solution for environment-dependent hotkeys. Phew.